DocAI Incubator Tools/best-practices/HITL_Rejected_Documents_Tracking.ipynb

```diff
@@ -0,0 +1,415 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    from google.cloud import documentai_v1beta3 as documentai
</pre><p>Is there a reason beta3 is used instead of v1?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>